RMextract/PosTools.py

```diff
@@ -1,42 +1,49 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+import os
+from math import atan, atan2, cos, degrees, pi, radians, sin, sqrt, tan
+
+import astropy.units as u
+import numpy as np
+from astropy.coordinates import (FK5, ITRS, AltAz, Angle, EarthLocation,
+                                 SkyCoord)
+from astropy.time import Time
+from scipy import interpolate
+
+from RMextract.logging import logger
+
 HAS_PYRAP = True
 try:
-  from pyrap import tables as tab
-  import pyrap.quanta as qu
-  from pyrap.measures import measures
-  print ('pyrap will be used to compute positions')
-except:
-  print ('We will need PyEphem to perform calculations!')
-  print ('the accuracy of results might decease a bit')
-  HAS_PYRAP = False
+    import casacore.quanta as qu
+    from casacore import tables as tab
+    from casacore.measures import measures
+    logger.debug('casacore will be used to compute positions')
+except ImportError:
+    logger.debug('We will need PyEphem to perform calculations!')
+    logger.debug('the accuracy of results might decease a bit')
+    HAS_PYRAP = False
 
 HAS_EPHEM = True
 try:
-  import ephem
-  if not HAS_PYRAP:
-    print ('PyEphem will be used to perform calculations!')
-except:
-  HAS_EPHEM = False
-
-from math import *
-import math
-import os
-import numpy as np
-import astropy.units as u
-from astropy.time import Time
-from astropy.coordinates import SkyCoord, EarthLocation, AltAz,Angle,ITRS,FK5
-from scipy import interpolate
+    import ephem
+    if not HAS_PYRAP:
+        logger.debug('PyEphem will be used to perform calculations!')
```

Comment:
I put these at debug level since they occur on import. Previously, for any dependent module (e.g. [FRion](https://github.com/CIRADA-Tools/FRion)) or a multiprocessing invocation these messages will appear _a lot_.